platform/darwin/docs/guides/Predicates and Expressions.md

```diff
@@ -152,6 +152,17 @@ NSExpression(forFunction: "lowercase:",
              arguments: [NSExpression(forKeyPath: "ISO 3166-1:2006")])
 ```
 
+If a feature attribute name contains a dot, the string can be wrapped in a
```

Comment:
Nit: replace “dot” with “period”. Also, avoid passive voice: “wrap the string in”.